api/v1beta3/vcdclustertemplate_conversion.go

```diff
@@ -0,0 +1,5 @@
+package v1beta3
+
+func (*VCDClusterTemplate) Hub() {}
+
+func (*VCDClusterTemplateList) Hub() {}
```

Comment:
We need a conversion webhook to convert resources from v1beta2 to v1beta3 ?

Reply:
A webhook for vcdclustertemplate is already present (under `config/crd/patches/webhook_in_vcdclustertemplates.yaml`)

Reply:
For some reason, I don't see this patch applied on the generated CRD